pkg/flags/flags.go

```diff
@@ -19,6 +22,7 @@ type CommonFlags struct {
 	SkipPrompt           bool
 	SkipVerify           bool
 	Profile              string
+	Subdomain            string
```

Comment:
Ideally, I don't want to put these 4 new fields there. They are only needed for OneLogin. Any ideas where to put them? I wanted to refactor the library and allow each provider to have custom config and secrets (same applies for `ClientID` and `ClientSecret` [here](https://github.com/Versent/saml2aws/pull/200/files#diff-d589c31a93f96430bf47cb898695396aR8).